crimson/fusioncatcher.py

```diff
@@ -166,15 +167,17 @@ def parse(in_data):
         first_line = src.readline().strip()
         # Parse column names
         colnames = first_line.split("\t")
-        if colnames == _COLS_0_99_5a:
-            COLS = _COLS_0_99_5a
-        elif colnames == _COLS_1_00:
-            COLS = _COLS_1_00
+
+        # Determine the version of fusioncatcher based on the column names
+        for version in _COLS:
+            if colnames == _COLS[version]:
+                break
```

Comment:
Hmm..something like this be cleaner instead? I find expressions (e.g. comprehensions) are often better to convey the intent of the code, as opposed to statements (e.g. for-loops).

```
# If none of the colnames match the values in _COLS
if not any(colnames == cols for cols in _COLS.values()):
    msg = "Unexpected column names: {0}."
    raise click.BadParameter(msg.format(colnames))
```

For this part, it seems the `_COLS` lookup is not necessary anymore since at this point we already know that `colnames` must be equal to one of the values in `_COLS`, right?
```
for line in src:
    parsed = parse_raw_line(line, colnames)
    ...
```